lib/router.js

```diff
@@ -37,7 +38,7 @@ const FEATURES = new Map([
     routes: [{
       name: 'home',
       path: routeConfig.path,
-      beforeEnter: requireLogin,
+      beforeEnter: [requireLogin, notifyAllowWallet],
```

Comment:
We discussed moving this elsewhere -- just noting that I know this bit will change.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vue-wallet/pull/29/commits/afac71d291de0810cb8527f5e19612374c2adddb